src/transactionComputer.ts

```diff
@@ -116,5 +131,15 @@ export class TransactionComputer {
         if (!transaction.chainID.length) {
             throw new errors.ErrBadUsage("The `chainID` field is not set");
         }
+
+        if (transaction.version < TRANSACTION_VERSION_DEFAULT) {
```

Comment:
Tricky if we will have, at some point, a default of 3.

`TRANSACTION_MIN_VERSION_THAT_SUPPORTS_OPTIONS` might come in handy here (just an idea).

Reply:
added.